newdle/client/src/components/answer/Calendar.js

```diff
@@ -19,6 +19,7 @@ import {setAnswer, setAnswerActiveDate} from '../../actions';
 import DayCarousel from '../DayCarousel';
 import styles from './answer.module.scss';
 import {getUserTimezone} from '../../selectors';
+import classNames from 'classnames';
```

Comment:
If you want to start using it, explicitly `npm install --save classnames` inside `newdle/client` so it gets added to `newdle/client/package.json` and we don't just use it because something else already install it.

Reply:
I just saw your comment - I don't have a strong opinion on whether to use `classNames` or not

Reply:
I just pushed this change